src/Microsoft.Health.Dicom.Functions/BlobMigration/DeleteDurableFunction.Orchestration.cs

```diff
@@ -0,0 +1,82 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Dicom.Core.Features.Model;
+using Microsoft.Health.Dicom.Core.Models.BlobMigration;
+using Microsoft.Health.Dicom.Functions.Indexing.Models;
+using Microsoft.Health.Operations.Functions.DurableTask;
+
+namespace Microsoft.Health.Dicom.Functions.BlobMigration;
+
+public partial class DeleteDurableFunction
+{
+    /// <summary>
+    /// Asynchronously copy DICOM instances.
+    /// It goes through DICOM instances in the past, copy to files as new blob format.
+    /// </summary>
+    /// <param name="context">The context for the orchestration instance.</param>
+    /// <param name="logger">A diagnostic logger.</param>
+    /// <returns>A task representing the <see cref="DeleteFilesAsync"/> operation.</returns>
+    [FunctionName(nameof(DeleteFilesAsync))]
+    public async Task DeleteFilesAsync(
```

Comment:
Are the name for the orchestration and activity too generic? Should they indicate they are part of the the migration, as officially the name of the functions are like `DeleteFileAsync` and I suspect we'll have an async delete in the near-ish future.

Reply:
Yeah make sense, Changing to MigrateDeleteFilesAsync